### DISCUSSION TITLE
Diff in PR code review doesn't highlight changes

### DISCUSSION THREAD

**Discussion by JesperIRL** [community/community#65236] [Pull Requests]
### Select Topic Area

Bug

### Body

Sometimes in PR diffs the changes aren't highlighted. The old line has the dark read background, and the new line has the dark green background, but the differences between the lines aren't shown at all.

An example can be seen here: https://github.com/openjdk/guide/pull/97/files#diff-f43200945b93eacb14613755322fd170adbc7ba5905b9349859b23245f80249b

In this change there are two differences, a lower case c was changed to an upper case C, and the word "committer's" was changed to "Contributor's".

Not having those changes highlighted makes it very difficult to review the change, and it makes one wonder - what other changes aren't highlighted in other diffs?

I have seen this several times when editing markdown files. Is markdown not a supported filetype on GitHub? The reason I ask is when I look at the info about this particular project that contains at least 98% markdown, the "languages" meter says 63.1% makefiles, 33.2% CSS, and 3.7% HTML.

**Comment by wyattscarpenter** [comment]
I'm running into this lately and I agree that this failure is troubling and should be fixed.

It seems like maybe the word diff highlighter gives up if there's too much text. But, um, it shouldn't.

**Comment by ljramac** [comment]
Hey Jesper, I feel you—those tiny diffs can be super annoying. One thing that helps me is using a local diff tool or an editor that highlights changes line-by-line, especially for markdown. Sometimes GitHub’s built-in diff just doesn’t cut it for subtle stuff like capitalization or small word tweaks.

Also, if it keeps happening on markdown, maybe try splitting big edits into smaller commits—it makes the changes more obvious and easier to review. Hope that makes reviewing less painful!
